core/src/main/java/org/apache/accumulo/core/client/admin/TableOperations.java

```diff
@@ -1090,6 +1090,20 @@ default Stream<TabletInformation> getTabletInformation(final String tableName, f
     throw new UnsupportedOperationException();
   }
 
+  /**
+   * @param ranges the ranges of tablets to scan. Ranges can overlap and an attempt will be made to
+   *        merge this list
+   * @param fields can optionally narrow the data retrieved per tablet, which can speed up streaming
+   *        over tablets. If this list is empty then all fields are fetched.
+   * @return a stream of tablet information for tablets that fall in the specified ranges. The
+   *         stream may be backed by a scanner, so it's best to close the stream.
+   * @since 4.0.0
+   */
+  default Stream<TabletInformation> getTabletInformation(final String tableName,
```

Comment:
We could drop the other getTabletInformation method from the API, its a new 4.0 API so we do not have to keep it for any reason.

Would be good to eventually make this api take a `List<RowRange>` instead of `Range`.   We can only really use the row part of the range.

Reply:
Dropped the single Range method in 636b10b.

> Would be good to eventually make this api take a `List<RowRange>` instead of `Range`. We can only really use the row part of the range.

Yea I was thinking about this. I'll take another look to see if its quick to do here or if that should be a follow on task.

Reply:
Updated to use RowRange in 92ca459